packages/runner/src/studio/studio-modals.jsx

```diff
@@ -26,11 +27,11 @@ export class StudioInstructionsModal extends Component {
             </div>
             <div className='text center-box'>
               <ul>
-                <li><pre>cy.check()</pre></li>
-                <li><pre>cy.click()</pre></li>
-                <li><pre>cy.select()</pre></li>
-                <li><pre>cy.type()</pre></li>
-                <li><pre>cy.uncheck()</pre></li>
+                <li><pre>cy.get(...).check()</pre></li>
+                <li><pre>cy.get(...).click()</pre></li>
+                <li><pre>cy.get(...).select()</pre></li>
+                <li><pre>cy.get(...).type()</pre></li>
+                <li><pre>cy.get(...).uncheck()</pre></li>
```

Comment:
No prefix for child commands. 

```suggestion
                <li><pre>.check()</pre></li>
                <li><pre>.click()</pre></li>
                <li><pre>.select()</pre></li>
                <li><pre>.type()</pre></li>
                <li><pre>.uncheck()</pre></li>
```